README.md

```diff
@@ -122,7 +122,7 @@ The node has some optional launch parameters that can be specified from the comm
 - `namespace` : namespace that the driver will run in. All services and publishers will be prepended with this, default: `/`
 - `node_name` : name of the driver, default: `microstrain_inertial_driver`
 - `debug`     : output debug logs, default: `false`
-- `params_file` : path to a parameter file to override the default parameters stored in [`params.yml`](./microstrain_inertial_driver/microstrain_inertial_driver_common/config/params.yml), default: empty
```

Comment:
I didn't realize this was broken. Thanks for fixing this. I would change this to use

https://github.com/LORD-MicroStrain/microstrain_inertial_driver_common/blob/main/config/params.yml

instead so it tracks the branch instead of the commit, otherwise looks good

Reply:
No problem! Ian and I saw it this morning. I am fairly new to Github still, is there anything I need to do to approve your comments?